src/components/hero/hero.scss

```diff
@@ -9,7 +9,7 @@
         }
     }
 
-    h1, h4 { color: $white; text-align: center; }
+    h1, h4 { color: $white; text-align: center; text-shadow: 5px 5px 10px $dark; }
```

Comment:
Seems like h1-h4 for hero have the same css properties now, maybe combine them/var them?

Reply:
I noticed this too, I'll refactor this in a future update